ChangeLog

```diff
@@ -1,3 +1,8 @@
+2019-11-18  Kun Ren  <mail@renkun.me>
+
+	* src/attributes.cpp: Add invisible param to export attribute (#1024)
+        * vignettes/rmd/Rcpp-attributes.Rmd: Add an section of Returning invisible object
```

Comment:
There is a minor misalignement here but do not worry about it.  I will probably make a new minor revision after the merge and can address that then.  Thanks for taking care of the `ChangeLog` entries.

Reply:
Sorry for forgetting about the second tab here.